assets/js/blocks/cart-checkout/cart/index.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { InnerBlocks } from '@wordpress/editor';
+import { registerBlockType } from '@wordpress/blocks';
+
+/**
+ * Internal dependencies
+ */
+import Block from './block';
+
+/**
+ * Register and run the "Featured Product" block.
+ */
+registerBlockType( 'woocommerce/cart', {
+	title: __( 'Cart', 'woo-gutenberg-products-block' ),
+	icon: {
+		src: 'cart',
+		foreground: '#96588a',
+	},
+	category: 'woocommerce',
+	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
+	description: __( 'Shopping cart.', 'woo-gutenberg-products-block' ),
+	supports: {
+		align: [ 'wide', 'full' ],
+		html: false,
+	},
+	attributes: {},
+
+	/**
+	 * Renders and manages the block.
+	 */
+	edit( props ) {
+		return <Block { ...props } />;
```

Comment:
We will end up moving this to a separate `Edit` component later that will call `<Block>` inside it with a `<Disabled>` like other blocks
https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/master/assets/js/blocks/active-filters/edit.js#L84